lib/core/src/irods_path_recursion.cpp

```diff
@@ -323,25 +321,16 @@ irods::file_system_sanity_check( irods::recursion_map_t& pathmap,
             return USER_INPUT_OPTION_ERR;
         }
 
-        if (DISPLAY_PREFLIGHT) { std::cout << "Running recursive pre-scan... " << std::flush; }
```

Comment:
Oh, right, we had discussed making this appear when "verbose" is in use. I can add this if desired.

Reply:
i think that makes sense

Reply:
Done